girderformindlogger/web_client/src/views/body/UserAccountView.js

```diff
@@ -27,6 +27,7 @@ var UserAccountView = View.extend({
             this.$('#g-user-info-error-msg').empty();
 
             var params = {
+                login: this.$('#g-login').val(),
```

Comment:
This is good but unnecessary. This interface is vestigial from the upstream Girder project and not one in use for MindLogger.